server/resolvers/TestPlanVersionOperations/updatePhaseResolver.js

```diff
@@ -253,6 +253,68 @@ const updatePhaseResolver = async (
         throw new Error('No test plan reports found.');
     }
 
+    if (phase === 'CANDIDATE') {
+        const hasChromeJaws = testPlanReports.find(
+            ({ browser, at }) => browser.id == 2 && at.id == 1
+        );
+        const hasChromeNVDA = testPlanReports.find(
+            ({ browser, at }) => browser.id == 2 && at.id == 2
+        );
+        const hasSafariVoiceOver = testPlanReports.find(
+            ({ browser, at }) => browser.id == 3 && at.id == 3
+        );
+
+        const hasRequiredCandidateReports =
+            hasChromeJaws && hasChromeNVDA && hasSafariVoiceOver;
+
+        if (!hasRequiredCandidateReports) {
+            const missingCombinations = [
+                hasChromeJaws ? '' : 'Chrome and JAWS',
+                hasChromeNVDA ? '' : 'Chrome and NVDA',
+                hasSafariVoiceOver ? '' : 'Safari and VoiceOver'
+            ].flatMap(str => str);
```

Comment:
```suggestion
            ].filter(str => str);
```

You may have wanted to use `filter` based on the truthy check callback? `flatMap` isn't doing anything here. 

Otherwise, it could end up with `missingCombinations` looking like `[ '', 'Chrome and JAWS', 'Safari and VoiceOver' ]` if reports exist for some of the combinations but not all. That would result in the error string displaying: `... have not been collected: , Chrome and NVDA, Safari and VoiceOver`, where there's a stray comma beside empty space due to the `.join()`.

Reply:
Hey, @howard-e, you are 100% correct here. This has been fixed in the most recent build.